app-api/src/test/java/org/openmrs/eip/app/route/sender/SenderSyncMsgReaderRouteTest.java

```diff
@@ -73,6 +73,8 @@ public void shouldLoadSenderSyncMessagesSortedByDateCreatedAndCallTheMsgProcesso
 		assertEquals(msgCount, msgs.size());
 		assertTrue(msgs.get(0).getDateCreated().getTime() > (msgs.get(2).getDateCreated().getTime()));
 		assertTrue(msgs.get(1).getDateCreated().getTime() > (msgs.get(2).getDateCreated().getTime()));
+		assertTrue(msgs.get(0).getDateSent().getTime() < (msgs.get(2).getDateSent().getTime()));
```

Comment:
Why did you add these? There is no requirement to sort the events by date sent like we do for date created